pkg/binding/transcoder/add_or_update_metadata.go

```diff
@@ -0,0 +1,20 @@
+package transcoder
+
+import (
+	"github.com/cloudevents/sdk-go/pkg/binding"
+	"github.com/cloudevents/sdk-go/pkg/binding/spec"
+)
+
+func AddOrUpdateAttribute(attribute spec.Kind, defaultValue interface{}, updater func(interface{}) (interface{}, error)) []binding.TransformerFactory {
```

Comment:
`Add or Update` seems like it means `Set`

Reply:
Could it be confused with `Add`?

Reply:
I would say drop Add and Update and only have Set, you can set nil or a none type to remove. No need for Add and Update overall